ognjen/pay-pal/src/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
You are importing bootstrap twice.
https://github.com/pvukmanovic-pannonians/pannonians-obuka/pull/91/files#diff-2dadadde3210084a25d3b51471d49b4787cd26530f19c0b6383892b35c3fd757R3